python-checks/src/test/java/org/sonar/python/checks/AllBranchesAreIdenticalCheckTest.java

```diff
@@ -45,30 +45,31 @@ public void quickfix_one_statement() {
       "    doSomething()\n";
```

Comment:
I think I didn't see a test for the problematic code showed in the ticket? 
More specifically about point 3)
> The following statement do_something() gets pushed outside the scope of the function.